core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.spec.ts

```diff
@@ -169,6 +170,16 @@ describe('Subtopic editor tab', () => {
     expect(component.editableTitle).toEqual('Subtopic1');
   });
 
+  it('should assign default values to modal when initialized', () => {
```

Comment:
I don't think this test brings value. It's a copy-paste of the constructor, it will not detect introduced bugs

Reply:
I have removed it for now, since, the tests would only be checking if the assignment operation in JavaScript works correctly.
Do we really need a test here? If so, what would be a good test for this? 

Thanks

Reply:
Constructors usually don't require tests, so no, no need for a test here